src/uu/stat/src/stat.rs

```diff
@@ -93,9 +97,30 @@ pub enum OutputType {
     Unknown,
 }
 
+enum QuotingStyle {
+    Locale,
+    Shell,
+    Default,
+    Quote,
+}
+
+impl std::str::FromStr for QuotingStyle {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "locale" => Ok(QuotingStyle::Locale),
+            "shell" => Ok(QuotingStyle::Shell),
```

Comment:
See my previous comment.
```suggestion
            "shell" => Ok(QuotingStyle::Shell),
            "shell-escape-always" => Ok(QuotingStyle::ShellEscapeAlways),
```